### ISSUE TITLE
Feature Request: Custom fonts 

### ISSUE THREAD

**Issue by hghgthifg** [jcv8000/Codex#64]
![图片](https://github.com/jcv8000/Codex/assets/61531662/64be6112-1746-4abf-8f54-6ab8bc716582)

I need more options than just the ones listed now

**Comment by HomeHeartTherapy** [comment]
Yeah.  Just installed the latest version. Looks BEAUTIFUL ❤️ Thank you, Josh!  I'd love to be able to use any font that is loaded on the system.  Would be amazing!
